bundles/action/org.openhab.action.xmpp/README.md

```diff
@@ -4,37 +4,38 @@ This set of actions allows you to send a message to an XMPP user or multi-user c
-| password | | | The password of the XMPP account used by openHAB |
-| servername | | | The XMPP service to use, e.g. `jabber.de`. A list of public XMPP services can be found at https://xmpp.net/directory.php |
-| securitymode | `disabled` | No | The Security mode used for the XMPP connection. Can be either `required` or `disabled`. Defaults to `disabled`, which means that TLS will not be used.  Warning: If you change this to non-disabled, then you must make sure that your TLS server certificate can be validated, otherwhise the connection will fail. |
-| tlspin | | | The TLS Pin used to verify the XMPP service's certificate. Set this in case openHAB's default `SSLContext` is unable to verfiy it (e.g. because the XMPP service uses a self-signed certificate). The PIN value is bascially the hash of the certificate in hexadecimal notation. You have to set `securitymode` to `required` to enable TLS for XMPP connections. For information on how to generate the PIN visit https://github.com/Flowdalic/java-pinning .  Example: `tlspin=CERTSHA256:83:F9:17:1E:06:A3:13:11:88:89:F7:D7:93:02:BD:1B:7A:20:42:EE:0C:FD:02:9A:BF:8D:D0:6F:FA:6C:D9:D3`
-| proxy | | | The XMPP Proxyserver to use, e.g. `gmail.com` |
-| port | `5222` | | The server port to use |
-| consoleusers | | | A comma-separated list of users that are allowed to use the XMPP console |
-| chatroom | | | The multi user chat to join, e.g. `openhab@chat.example.com` |
-| chatnickname | `openhab-bot` | No | The nickname used in the multi-user chat |
-| chatpassword | | | The password required to join the multi user chat |
+| Property     | Default       | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|--------------|---------------|---------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

```

Comment:
Can you type in the `:` by hand?

So we get from
`|---------:|`
to
`|:---------:|`

I think that's the only thing thats missing here.